src/main/java/uk/gov/hmcts/reform/iacasenotificationsapi/domain/utils/AsylumCaseUtils.java

```diff
@@ -239,6 +239,13 @@ public static List<String> getAppellantAddressAsList(final AsylumCase asylumCase
         return appellantAddressAsList;
     }
 
+    public static boolean hasAppellantAddressInCountryOrOoc(AsylumCase asylumCase) {
```

Comment:
@Ryan-Nesbitt , Jamie added this yesterday. Last method of the class.

Reply:
Thanks Thomas, I removed mine now and used Jamie's method.